core/src/main/java/org/ehcache/spi/cache/Store.java

```diff
@@ -441,7 +441,7 @@
      * Informs this Provider, a Store it created is being disposed (i.e. closed)
      * @param resource
      */
-    void releaseStore(Store<?, ?> resource);
+    void releaseStore(Store<?, ?> resource) throws CacheAccessException;
```

Comment:
I'm not sure rethrowing CacheAccessException is the right thing to do here. IMHO the releaseStore() implementations should handle those internally.